src/Libraries/RevitNodes/Elements/StucturalFraming.cs

```diff
@@ -247,17 +247,28 @@ private void InternalSetCurve(Autodesk.Revit.DB.Curve crv)
         /// <summary>
         /// Gets curve geometry from location of the specified structural element
         /// </summary>
-        new public Autodesk.DesignScript.Geometry.Curve Location
+        public new Autodesk.DesignScript.Geometry.Geometry Location
```

Comment:
Mhmmm...good point. Should we update the `AbstractFamilyInstance` base implementation of the `Location` property instead? That will cover this example, but will open up a few more fixes in other classes that inherit from it...possibly.